packages/cactus-cmd-api-server/src/test/typescript/integration/jwt-endpoint-authz-scope-enforcement.test.ts

```diff
@@ -142,22 +139,16 @@ test(testCase, async (t: Test) => {
           Authorization: `Bearer ${token}`,
         },
       });
-      t.fail("deploy contract response status === 403 FAIL");
-    } catch (out) {
-      t.ok(out, "error thrown for forbidden endpoint truthy OK");
-      t.ok(out.response, "deploy contract response truthy OK");
-      t.equal(
-        out.response.status,
-        StatusCodes.FORBIDDEN,
-        "deploy contract response status === 403 OK",
-      );
-      t.notok(out.response.data.data, "out.response.data.data falsy OK");
-      t.notok(out.response.data.success, "out.response.data.success falsy OK");
+    } catch (out: unknown) {
+      if (axios.isAxiosError(out)) {
+        expect(out).toBeTruthy();
+        expect(out.response).toBeTruthy();
+        expect((out as any).response.status).toBe(StatusCodes.FORBIDDEN);
```

Comment:
@Leeyoungone Please no casting to `any`. You can use the `toHaveProperty` assertions instead: https://stackoverflow.com/a/47904961

Reply:
Got it! I've refactored so that the test for the status code uses the `toHaveProperty`, however there is a way to also use `toBe` and the `StatusCodes.FORBIDDEN` if we wanted to keep the code consistent with what we've had before. Below, I've listed the two ways to do this 


-- using the `toBe` and `StatusCodes.FORBIDDEN`

``` Typescript
expect(out.response?.status).toBe(StatusCodes.FORBIDDEN);
```


-- using the `toHaveProperty`
``` Typescript
expect(out).toHaveProperty(["response", "status"], 403);
```

Please let me know which one you'd prefer! For now I'm using the `toHaveProperty` because of your suggestion above.